src/components/home/navbar/DropdownButton.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+
+import DropdownMenu from './DropdownMenu'
+
+import '../../../styles/home/navbar/dropdown_button.css'
+
+class DropdownButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isOpen: false
+    }
+    this.toggleMenu = this.toggleMenu.bind(this)
+    this.onClose = this.onClose.bind(this)
+  }
+
+  toggleMenu (event) {
```

Comment:
Remove space between function name and parens.
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```